src/Modules/UngradedModules.jl

```diff
@@ -154,30 +165,99 @@ end
     FreeModElem{T}
 
 The type of free module elements. A free module element
-is determined by a sparse row (`SRow`) which contains the coords 
-with respect to the standard basis and the parent free module.
+is given by a sparse row (`SRow`) which specifies its coordinates 
+with respect to the basis of unit vectors.
 
-# Example
+# Examples
 ```jldoctest
-julia> R, (x,y) = PolynomialRing(QQ, ["x", "y"])
+julia> R, (x, y) = PolynomialRing(QQ, ["x", "y"])
 (Multivariate Polynomial Ring in x, y over Rational Field, fmpq_mpoly[x, y])
 
-julia> F = FreeMod(R,2)
-Free module of rank 2 over Multivariate Polynomial Ring in x, y over Rational Field
+julia> F = FreeMod(R, 3)
+Free module of rank 3 over Multivariate Polynomial Ring in x, y over Rational Field
 
-julia> v = FreeModElem(sparse_row(R, [(1,x),(2,y)]),F)
-(x)*e[1] + (y)*e[2]
+julia> f = FreeModElem(sparse_row(R, [(1,x),(3,y)]),F)
+(x)*e[1] + (y)*e[3]
 
-julia> v == x*F[1]+y*F[2]
-true
+julia> g = (x)*F[1] + (y)*F[3]
+(x)*e[1] + (y)*e[3]
 
+julia> typeof(g)
+FreeModElem{fmpq_mpoly}
+
+julia> f == g
+true
 ```
 """
 struct FreeModElem{T} <: AbstractFreeModElem{T}
   coords::SRow{T} # also usable via coeffs()
   parent::FreeMod{T}
+
+  function FreeModElem{T}(coords::SRow{T}, parent::FreeMod{T}) where T
+    r = new{T}(coords,parent)
+    return r
+  end
+end
+
+@doc Markdown.doc"""
+    FreeModElem(c::SRow{T}, parent::FreeMod{T}) where T
+
+Return the element of  `F`  whose coefficients with respect to the basis of unit vectors
```

Comment:
You keep repeating "the basis of unit vectors". I assume this is referring to some "canonical basis", yes? Then I'd state it more like that... I find the reference to "unit vectors" otherwise a bit weird?